src/providers/wms/CMakeLists.txt

```diff
@@ -46,6 +46,8 @@ INCLUDE_DIRECTORIES(SYSTEM
 ADD_LIBRARY(wmsprovider_a STATIC ${WMS_SRCS} ${WMS_MOC_SRCS})
 ADD_LIBRARY(wmsprovider MODULE ${WMS_SRCS} ${WMS_MOC_SRCS})
 
+ADD_DEPENDENCIES(wmsprovider qgis_gui)
```

Comment:
This should be inside a `IF (WITH_GUI)` block, right?

Reply:
I don't think so, the related targets are not within `IF (WITH_GUI)` blocks either.

Reply:
Ah - totally missed that this was for 2.18, sorry.